RecoLocalTracker/SiStripZeroSuppression/plugins/SiStripBaselineComparator.cc

```diff
@@ -112,9 +112,9 @@ void
 SiStripBaselineComparator::fillDescriptions(edm::ConfigurationDescriptions & descriptions) {
 
   edm::ParameterSetDescription desc;
-  desc.add<edm::InputTag>("srcClusters",edm::InputTag("siStripClusters",""));
-  desc.add<edm::InputTag>("srcClusters2",edm::InputTag("siStripClusters",""));
-  descriptions.addDefault(desc);
+  desc.add<edm::InputTag>("srcClusters",  edm::InputTag("siStripClusters", "siStripClusters"));
```

Comment:
this doesn't look right, I think that the siStripClusters branch has no instance defined.
So, it should just be ```edm::InputTag("siStripClusters")```.
I guess ```srcClusters2``` is a placeholder, because afaik ```moddedsiStripClusters``` does not exist in the release.